Launching-new-features.md

```diff
@@ -38,43 +38,49 @@ Say you are working on a large scale user-facing feature that will take 1 or mor
 
 9. Once the feature is confirmed to be functioning as intended in production (for at least 2 weeks) by the product team, please do the following, in order:
     - Make sure that the feature is ready to be made permanent. To do this, confirm with the PMs that no users have reported issues with it, and that no regressions have been detected via StackDriver or general user feedback. The PMs should also fill in this [post-launch review template](https://docs.google.com/document/d/1DifFAe3oRzjmVPh2fEllfAky4n0QMAXVQc3Y580qkr8/edit).
-    - Once you have confirmation that the feature can be made permanent, merge one last PR to "un-gate" the feature and move the feature flag to the deprecated stage (one of the stages listed in `core/domain/platform_feature_list.py`, meant for flags that are no longer in use). Additionally, in the same PR, please remove all remaining references to the feature flag from the codebase (for example, in all the `if` blocks you created to gate the feature).
+    - Once you have confirmation that the feature can be made permanent, merge one last PR to "un-gate" the feature and move the feature flag to the deprecated stage (one of the stages listed in `core/feature_flag_list.py`, meant for flags that are no longer in use). Additionally, in the same PR, please remove all remaining references to the feature flag from the codebase (for example, in all the `if` blocks you created to gate the feature).
 
 
 ## Follow the steps below to add a new feature flag
 
 ### Creating the flag
 
-1. Add a new unique feature flag name in the `PARAM_NAMES` enum class in `core/domain/platform_parameter_list.py`, similar to a key-value pair. Example:
+1. Add a new unique feature flag name in the `FeatureNames` enum class in `core/feature_flag_list.py`, similar to a key-value pair. Example:
 
 ```python
-class ParamNames(enum.Enum):
-    """Enum for parameter names."""
+class FeatureNames(enum.Enum):
+    """Enum for feature flag names."""
     // ...
     NEW_FEATURE = 'new_feature',
 
 ```
 
-2. Create and register a feature flag instance in the `core/domain/platform_parameter_registry.py` file with its name, description and stage (see the [Feature Stage](#feature-stage-explanation) section for more details). Example:
+2. Add the name of the new feature flag to one of the feature name lists (`DEV_FEATURES_LIST`, `TEST_FEATURES_LIST`, or `PROD_FEATURES_LIST`) in `core/feature_flag_list.py` according to its stage. Note: if the name is added to the wrong list, a backend test error will raise. Example:
```

Comment:
Could you tell me a bit more about this error please? E.g. how would the backend tests know whether the name is added to the wrong list, since this depends on external info like feature testing results?

Reply:
Oh, I missed out on that. I have removed this part, actually, we cannot really verify that. Thanks for catching it!